anchor-cli.nix

```diff
@@ -37,27 +38,16 @@ rustPlatform.buildRustPackage rec {
     owner = "coral-xyz";
     repo = "anchor";
     rev = "v${version}";
-    hash = "sha256-3fLYTJDVCJdi6o0Zd+hb9jcPDKm4M4NzpZ8EUVW/GVw=";
+    hash = "sha256-CaBVdp7RPVmzzEiVazjpDLJxEkIgy1BHCwdH2mYLbGM=";
   };
 
   cargoLock = {
     lockFile = "${src.outPath}/Cargo.lock";
     allowBuiltinFetchGit = true;
   };
 
-  patches = [
-    ./anchor-cli.patch
-  ];
-
   buildAndTestSubdir = "cli";
 
-  # Ensure anchor has access to Solana's cargo and rust binaries
-  postInstall = ''
-    rust=${solana-platform-tools}/bin/sdk/sbf/dependencies/platform-tools/rust/bin
-    wrapProgram $out/bin/anchor \
-      --prefix PATH : "$rust"
-  '';
```

Comment:
would anchor use the correct rust version without this wrap?

Reply:
The Anchor CLI creates a subcommand to execute `cargo build-sbf`, which uses the correct version of Rust, so we're all good!

I built and deployed a program build with `anchor build` to make sure.

Reply:
Yep that's correct, just `build`